spec/frameSpec.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+describe('Frame', function () {
+  var frame;
+  var scorecard;
+
+  describe('when a bowler throws two non-bonus bowls', function () {
+  
+    beforeEach(function () {
+      frame = new Frame();
+      frame.addBowlOneScore(7);
+      frame.addBowlTwoScore(2);
+    });
+
+    it('calculates the first bowl score', function () {
+      expect(frame.bowlOne).toEqual(7);
+    });
+
+    it('calculates the second bowl score', function () {
+      expect(frame.bowlTwo).toEqual(2);
+    });
+
+    it('calculates the total of two bowls', function () {
+      expect(frame.calculateTotal()).toEqual(9);
+    });
+    
+  });
+
+  describe('when a bowler throws a 10', function () {
```

Comment:
Does 'when a bowler throws a 10' applies to all of the test cases within this `describe` block?